<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialView</class>
 <widget class="QMainWindow" name="SerialView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ximu::SerialView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc/serialview.qrc">
    <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="textArea"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBoxBaudrates">
      <property name="title">
       <string>Connection</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Com:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxPortNames</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Baudrate:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxBaudRates</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="5" rowspan="2">
        <widget class="LanguageSelectionWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <zorder>horizontalSpacer</zorder>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxPortNames"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxBaudRates"/>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>38</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButtonOnOff">
         <property name="text">
          <string>open</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>87</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LanguageSelectionWidget</class>
   <extends>QWidget</extends>
   <header>include/ximugui/widgets/lanuageselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/serialview.qrc"/>
 </resources>
 <connections/>
</ui>
